contracts/crowdsale/ZxcCrowdsale.sol

```diff
@@ -243,51 +241,24 @@ contract ZxcCrowdsale is
   }
 
   /**
-   * @dev Check if currently active period is pre-sale with bonuses.
+   * @dev Check if currently active period is a given time period.
+   * @param _startTime Starting timestamp (inclusive).
+   * @param _endTime Ending timestamp (exclusive).
    * @return bool
    */
-  function isPresale()
+  function isInTimeRange(uint256 _startTime, uint256 _endTime)
```

Comment:
Code style: each parameter should be in its own line.

Reply:
ee941b6